config/authentication.go

```diff
@@ -0,0 +1,17 @@
+package config
```

Comment:
I don't think this belongs in the `config` package. Can it be moved to `handlers`?

Reply:
I think there is already an `authentication.go` in `handlers`.  Perhaps this can be merged in?   Or it can stay here if you think it makes more sense, but I agree that I don't currently see why this is config.

Reply:
Well the point is that we need to have this in the same file I don't care about where I thought in the config by the "authInfo" variable let me merge this in handlers

Reply:
mmmm we'll have import cycle if merge this in handlers/authentication